extensions/catalogs/managing-catalogs.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-catalogs"]
+= Managing catalogs
+include::_attributes/common-attributes.adoc[]
+:context: managing-catalogs
+
+toc::[]
+
+Cluster administrators can add _catalogs_, or curated collections of Operators and Kubernetes extensions, to their clusters. Operator authors publish their products to these catalogs. When you add a catalog to your cluster, you have access to the versions, patches, and over-the-air updates of the Operators and extensions that are published to the catalog.
+
+You can manage catalogs and extensions declaratively from the CLI using custom resources (CRs).
```

Comment:
```suggestion
You can manage catalogs and extensions declaratively from the CLI by using custom resources (CRs).
```
Tiny change to avoid ambiguity.